### DISCUSSION TITLE
7 settembre 2023

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/mdp2223#9] [Q&A] [answered]
In questa discussion a tema la traccia di esame del 7 settembre 2023 discuteremo di eventuali proposte di soluzione all'esame in questione

**Comment by Gio0203c** [comment, marked as answer]
[07_09_2023 - COMPLETO.pdf](https://github.com/sapienzastudentsnetwork/mdp2223/files/12988625/07_09_2023.-.COMPLETO.pdf)

**Comment by FedericoSte** [comment]
Ciao, volevo domandarti se per caso sapessi dirmi il nome del docente che c'era negli anni 2021/2022 - 2022/2023.
Sono iscritto al 1° anno di Informatica (canale A-L) e nel corso di Metodologie di Programmazione il docente è Mattia Samory, però da quel che ho capito è il primo anno che c'è lui, non vorrei che stravolga lo scritto (gli scritti che hai pubblicato mi sembrano rispettare tutti più o meno lo stesso Pattern).
Ps. Se qualcuno dovesse avere lo scritto che c'è stato il 18/06/2024 sarebbe un grande salvatore.

**Reply by matypist** [reply]
Negli A.A. 2021-2022 e 2022-2023 i docenti erano il prof. Gabriele Etta e il prof. Walter Quattrociocchi

Sì, molto probabilmente l'esame adesso sarà ben diverso

**Reply by FedericoSte** [reply]
Non c'era il Docente Stefano Faralli?

**Reply by matypist** [reply]
Il prof. Faralli era assegnato come docente al canale 2 @FedericoSte , credevo ci stessimo riferendo al canale A-L

**Reply by matypist** [reply]
E questo repository in cui ci troviamo riguardava soltanto il canale A/L

**Comment by FedericoSte** [comment]
Okok perfetto, era solo per capire, dato che cercando al volo online mi è apparso più volte il nome del docente Faralli
